engineyard/ey-cookbooks-stable-v6
# Merge pull request #1 from engineyard/next-release

Hi there, I hope this works, it looked to be oriented more for internal stakeholders. Essentially I wanted the ability to have my logs default to syslog formatted, so I could then configure syslog-ng to parse my logs and send them up to a centralized system.

Description of your patch
-------------
This is a small patch to enable the pretend log paths with syslog to ensure syslog syntax is used.

Recommended Release Notes
-------------
Added support for syslog formatted logs

Estimated risk
-------------
No risk, feature requires opting in by customer, so default behavior remains.

Components involved
-------------
nginx, syslog

Dependencies
-------------
nginx

Description of testing done
-------------
Ran the app erb file locally and it generated the expected syntax.

QA Instructions
-------------
Include the cookbook and set the attribute:
default['nginx']['syslog'] = true
which should then have nginx use syslog format.
